src/lzf/lzfP.h

```diff
@@ -144,7 +144,9 @@ using namespace std;
 # if defined (WIN32)
 #  define LZF_USE_OFFSETS defined(_M_X64)
 # else
-#  if __cplusplus > 199711L
+#  if MAC_OS_X_VERSION_MIN_REQUIRED >= MAC_OS_X_VERSION_10_8
```

Comment:
@bobrik: I didn't even look at that PR because of the title >_<. Isn't my commit safer as it leaves all existing logic in and fixes the problem?